pom.xml

```diff
@@ -431,9 +431,26 @@
                     <artifactId>license-maven-plugin</artifactId>
                     <version>2.0.0</version>
                 </plugin>
+                <plugin>
+                    <artifactId>exec-maven-plugin</artifactId>
+                    <version>1.6.0</version>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <executions>
+                        <execution>
+                            <id>Install git-secrets</id>
+                            <phase>initialize</phase>
+                            <goals>
+                                <goal>exec</goal>
+                            </goals>
+                            <configuration>
+                                <executable>bash</executable>
+                                <commandlineArgs>${project.parent.basedir}/scripts/install-git-hooks.sh</commandlineArgs>
```

Comment:
I'm a bit confused (also ignorant of how exec-maven-plugin works) - is this command running the install hooks command on this (the local CI) machine, or is the command being run on some Maven server somewhere in the cloud?

Reply:
Should be running locally (or on CI when checked out there)

Reply:
You can think of maven like the `setup.py` for Java packaging. It defines dependencies that need to be downloaded and allows you to control various stages of building your project (verify, install, package, etc). In this case, our settings should be set on whatever machine is running the maven build which will be developer machines and CI.